src/electron-forge-publish.js

```diff
@@ -14,6 +14,8 @@ import { getMakeOptions } from './electron-forge-make';
     .option('--auth-token', 'Authorization token for your publisher target (if required)')
     .option('--tag', 'The tag to publish to on GitHub')
     .option('--target [target]', 'The deployment target, defaults to "github"')
+    .option('--dry-run', 'Triggers a publish dry run which saves state and doesn\'t upload anything')
```

Comment:
I think at this point, we would need to make sure to be very explicit in the docs that dry-run only fakes the publish part and not the distributable generation (because I can't think of a better option name).